client/me/application-passwords/index.jsx

```diff
@@ -184,42 +159,39 @@ const ApplicationPasswords = createReactClass( {
 							this.clearNewApplicationPassword
 						) }
 					>
-						{ this.props.translate( 'Done' ) }
+						{ translate( 'Done' ) }
 					</FormButton>
 				</FormButtonsBar>
 			</Card>
 		);
-	},
+	}
 
-	renderApplicationPasswords: function() {
-		if ( ! this.props.appPasswordsData.get().length ) {
+	renderApplicationPasswords() {
+		const { appPasswords, translate } = this.props;
+		if ( ! appPasswords.length ) {
 			return null;
 		}
 
 		return (
 			<div className="application-passwords__active">
-				<FormSectionHeading>{ this.props.translate( 'Active Passwords' ) }</FormSectionHeading>
+				<FormSectionHeading>{ translate( 'Active Passwords' ) }</FormSectionHeading>
 				<ul className="application-passwords__list">
-					{ this.props.appPasswordsData.get().map( function( password ) {
-						return (
-							<AppPasswordItem
-								password={ password }
-								appPasswordsData={ this.props.appPasswordsData }
-								key={ password.ID }
-							/>
-						);
+					{ appPasswords.map( function( password ) {
+						return <AppPasswordItem password={ password } key={ password.ID } />;
 					}, this ) }
```

Comment:
I think we can drop the `this` argument here. Arrow for maximum brevity.

```js
appPasswords.map( password => <AppPasswordItem password={ password } key={ password.ID } /> )
```

Reply:
https://github.com/Automattic/wp-calypso/pull/23926/commits/734733c6b961917ad8254fc8fb86dcbff75da43f